docs/source/release-notes.rst

```diff
@@ -2,6 +2,22 @@ Release Notes
 =============
 
 
+Version 0.9.0
+_____________
+- Engine support to configure `AppConnections` and `EventConnections` to express App/Event dependencies.
+- Engine support for multiple client implementation via plugins
+
+- Plugins:
+  - New plugin: `hopeit.apps_client` allows invokation of other running apps via http.
+  - Apps Visualizer plugin: support for showing connections between connected Apps via AppConnections
+
+- BREAKING CHANGES:
+  - Engine `auth` module now create and store one pair of private/public keys per each running app. Keys are stored
```

Comment:
Engine `auth` module now creates and stores one pair of private/public keys per running app.